src/extension/background-script/CryptoServices/utils.ts

```diff
@@ -28,20 +31,43 @@ export interface TypedMessageText extends TypedMessageMetadata {
     readonly type: 'text'
     readonly content: string
 }
+export interface TypedMessageImage extends TypedMessageMetadata {
```

Comment:
Clearify: TypedMessage is used to describe the information _after_ the decryption. Please make sure it is used in this way.

Reply:
Of course, It's used in that way.